### ISSUE TITLE
Concave hull calculation?

### ISSUE THREAD

**Issue by ibrahimLearning** [JeremyBYU/polylidar#3]
I need to calculate the concave hull. I would like to use your repo to calculate due to faster than other packages.

Would you give a way to calculate concave hull using polylidar?
Thanks in advance

**Comment by JeremyBYU** [comment]
Polylidar returns a **list** of polygons. 

A polygon is composed of a **shell** and a _list_ of **holes**.

The shell **is** the concave hull of the polygon. It is an ordered linear ring of point indices. If you just want the shell then ignore the holes that are returned in the polygon.

Look at the basic2d example which shows how to visualize the polygon. Here is a link to where you can play around with this in the browser: https://mybinder.org/v2/gh/jeremybyu/polylidar/master?filepath=examples%2Fpython%2FBasicDemo.ipynb

Basically you want ‘’’polygon.shell’’’.

Hope that helps out. Note that if you just want a _single_ concave hull from 2d points there are faster methods. https://github.com/mapbox/concaveman

**Comment by ibrahimLearning** [comment]
> Polylidar returns a **list** of polygons.
> 
> A polygon is composed of a **shell** and a _list_ of **holes**.
> 
> The shell **is** the concave hull of the polygon. It is an ordered linear ring of point indices. If you just want the shell then ignore the holes that are returned in the polygon.
> 
> Look at the basic2d example which shows how to visualize the polygon. Here is a link to where you can play around with this in the browser: https://mybinder.org/v2/gh/jeremybyu/polylidar/master?filepath=examples%2Fpython%2FBasicDemo.ipynb
> 
> Basically you want ‘’’polygon.shell’’’.
> 
> Hope that helps out. Note that if you just want a _single_ concave hull from 2d points there are faster methods. https://github.com/mapbox/concaveman

Thanks for your reply. Yes I want single concave hull
